server/vm/base.ts

```diff
@@ -1,14 +1,21 @@
-import { Room } from '../room';
-import Redis from 'ioredis';
 import axios from 'axios';
+import Redis from 'ioredis';
```

Comment:
I'm not sure if these changes were automated or required because of crashes when REDIS_URL is not set.

If they're needed, maybe add them separately?